styles/theme.css

```diff
@@ -1370,4 +1370,8 @@ ul+p>img, ol+p>img {
 
 li>p:nth-child(n+2) {
     margin-top: 16px;
+}
+
+ais-highlight-0000000000 {
```

Comment:
is this used anywhere?

Reply:
Yes, it is the tag Algolia wraps matched search terms in
![image](https://user-images.githubusercontent.com/57426646/209759413-f59746f5-324d-4a89-a977-75c5488e19ba.png)

Reply:
please add comments so that its clear for other devs too

Reply:
Okay